plasmapy/particles/_factory.py

```diff
@@ -84,36 +175,22 @@ def _physical_particle_factory(
     # because they are not allowed as arguments to `CustomParticle`, and
     # are not needed in kwargs if they are their default values. Note
     # that this affects `not kwargs` below.
+
     for parameter in ("Z", "mass_numb"):
         if parameter in kwargs and kwargs[parameter] is None:
             kwargs.pop(parameter)
 
-    if (
-        len(args) == 1
-        and not kwargs
-        and isinstance(args[0], (Particle, CustomParticle, ParticleList))
-    ):
+    if len(args) == 1 and not kwargs and isinstance(args[0], _particle_types):
         return args[0]
 
     if not args and not kwargs:
         raise TypeError("Particle information has not been provided.")
 
-    for particle_type in (Particle, CustomParticle, ParticleList):
+    for constructor in _particle_constructors:
         with contextlib.suppress(TypeError, InvalidParticleError):
-            return particle_type(*args, **kwargs)
+            return constructor(*args, **kwargs)
```

Comment:
And you can still call the function above just like this, I think! It should just work out of the box without doing that weird "this needs a 1-tuple first argument" dance. Unless kwargs may have more than those two keyword arguments accepted by the long-named function above. If so, we'd have to sidestep that somehow.